virt/virtual_machines/advanced_vm_management/virt-specifying-nodes-for-vms.adoc

```diff
@@ -0,0 +1,32 @@
+[id="virt-specifying-nodes-for-vms"]
+= Specifying nodes for virtual machines
+include::modules/virt-document-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: virt-specifying-nodes-for-vms
+
+toc::[]
+
+You can place virtual machines (VMs) on specific nodes by using node placement rules.
+
+include::modules/virt-about-node-placement-vms.adoc[leveloffset=+1]
+
+[id="node-placement-virt-components_{context}"]
+=== About node placement for {VirtProductName} components
+* xref:../../../virt/install/virt-specifying-nodes-for-virtualization-components.adoc#virt-specifying-nodes-for-virtualization-components[Specifying nodes for virtualization components]
+
+[id="example-manifest-snippets_{context}"]
+== Example manifest snippets
```

Comment:
Suggested title: Node placement examples

Reply:
That is a much more specific title :)